api/src/queries/addressables.ts

```diff
@@ -0,0 +1,21 @@
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/types';
+
+export const addressablesQuery = (
+  queryString: string
+): QueryDslQueryContainer => ({
+  multi_match: {
+    query: queryString,
+    fields: [
+      'id',
+      'uid',
```

Comment:
I added `uid` which doesn't currently exist in the other queries. I wasn't sure whether we would want to add `type` here too (since it might help determine what we want to score higher amongst the various addressable types)